src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -23,346 +23,112 @@
 
   <SectionTitle title="Introduction" />
   <p>
-    In this guide we will explain how to build a new data source or productizer and how to use the
-    developer portal to publish it using the developer portal so it can be queried through the
-    product gateway using <A href="https://sandbox.ioxio-dataspace.com/">
-      IOXIO Sandbox Dataspace
-    </A>. These steps are the same on all IOXIO Dataspaces, while some might miss the particular
-    data products mentioned here.
+    In this guide we will explain how to build a new data source so it can be queried through the
+    product gateway using
+    <A href="https://sandbox.ioxio-dataspace.com/">IOXIO Sandbox Dataspace</A>. These steps are the
+    same on all IOXIO Dataspaces, while some might miss the particular data products mentioned here.
   </p>
   <SectionTitle title="Prerequisites" />
-  <li>Data that you want to provide.</li>
-  <li>A data source definition for the data.</li>
+  <li>Data that you want to provide available in a database.</li>
```

Comment:
```suggestion
  <li>Data that you want to provide available.</li>
```

The data can be in some file or read on demand from some sensor, it doesn't need to be in a database.